docs/_source/_common/snippets/start_page.md

```diff
@@ -1,36 +1,85 @@
-::::{tab-set}
+# Getting started with Argilla
 
-:::{tab-item} Feedback datasets
+Welcome to Argilla! 👋😁 We're thrilled to have you on board. Starting with Argilla is as simple as it gets, and you've successfully taken the first step!
+
+To start annotating your data, you need to create a `FeedbackDataset`. Let's set things in motion. 🚀
+
+[For a more in-depth understanding of Argilla, you can check our documentation 📖.](https://docs.argilla.io/en/latest/)
+
+
+## 1. Open an IDE, Jupyter or Collab
+
+1. Open your favorite IDE, Jupyter or Collab. You can also open our [introductory tutorial](https://colab.research.google.com/github/argilla-io/argilla/blob/develop/docs/_source/getting_started/quickstart_workflow_feedback.ipynb).
+
+2. Get your `API_URL`:
+
+   * If you are using Docker, it is the URL shown in your browser (by default `http://localhost:6900`)
+   * If you are using HF Spaces, it should be constructed as follows: `https://[your-owner-name]-[your_space_name].hf.space`
+
+3. Get your `API_KEY` by clicking on your user icon in the top right corner of the Argilla UI, selecting "My settings" and copying the API key.
+
+## 2. Install the SDK with pip
+
```

Comment:
To upload and read datasets from Argilla, you need to use the Argilla SDK. You can install the SDK with pip as follows: